app/assets/admin/js/settings.js

```diff
@@ -10,7 +10,7 @@ rtMediaAdmin.templates = {
 };
 
 jQuery( document ).ready( function ( $ ) {
-	
+
 	if ( $( '#rtm-licenses' ).length ) {
 		$( '#rtm-licenses .license-inner:first input:first' ).focus();
```

Comment:
Take one variable i.e. `var a = $( '#rtm-licenses' )` and then use its other properties
Use `a.find()` for its children.
@malavasita